css-animationworklet/Overview.bs

```diff
@@ -246,31 +340,27 @@ following steps:
 
     4. Let |prototype| be the result of <a>Get</a>(|animatorCtorValue|, "prototype").
 
-    5. If the result of <a>Type</a>(|prototype|) is not Object, <a>throw</a> a <a>TypeError</a>
-        and abort all these steps.
-
-    6. Let |animateValue| be the result of <a>Get</a>(|prototype|, "animate").
+    5. Let |type| be the the result of <a>Type</a>(|prototype|) .
```

Comment:
Will this actually return StatelessAnimator / StatefulAnimator, rather than 'FooAnimator' or whatever the class name is? I think it's pretty clear anyway, I was just wondering if there other existing wording for 'inherits from'.